tests/integration/s2n_tls13_handshake_tests.py

```diff
@@ -60,30 +60,58 @@ def verify_hrr_random_data(server, client):
 
     return result
 
+def send_and_receive(client, server, key_update_request):
```

Comment:
Nit: Can you specify in the name that this is for KeyUpdate? It sounds like a more general test